### DISCUSSION TITLE
Buzzer Type

### DISCUSSION THREAD

**Discussion by slepiavka** [danieleborgo/OctoPrint-ShutdownButtonLEDBuzzer#2] [Q&A] [answered]
Hi,

Does this need an active or passive buzzer?

Very cool project; simple is usually best.

**Comment by danieleborgo** [comment, marked as answer]
Hi,
I implemented an active buzzer but I can extend the code to support also passive ones. I just need to find one somewhere for testing.
Thank you

**Reply by slepiavka** [reply]
My unsolicited opinion is that it's not worth your time at all. I just think it's worthwhile specifying on the Wiki so people don't accidentally purchase the wrong buzzer and flood you with issues.
